### ISSUE TITLE
UUPDownload: please implement single-version fetch

### ISSUE THREAD

**Issue by lumag** [gus33000/UUPMediaCreator#227]
It would be nice to be able to fetch only a single version of the drivers (usually the latest one). Could you please consider implementing such mode in UUPDownload.

**Comment by gus33000** [comment]
Yeah that's considered but for a more standalone tool other than the current separate branch which really is more of a hack for the reference drivers repo. I got a lot of demand lately to make a standalone tool for this purpose and there is indeed some merit in doing so
